content/docs/operating/security.md

```diff
@@ -14,13 +14,15 @@ attack vectors that some configurations may enable.
 
 As with any complex systems it is not possible to guarantee that there are no
 bugs. If you find a security bug, please file it in the issue tracker of the
-relevant component.
+relevant component. If you prefer to report security issues in secret, you may
+email the non-public [Prometheus team mailing list](mailto:prometheus-team@googlegroups.com).
 
 ### Prometheus
 
 It is presumed that untrusted users have access to the Prometheus HTTP endpoint
 and logs. They have access to all time series information contained in the
-database, plus a variety of operational/debugging information.
+database, plus a variety of operational/debugging information, including all
```

Comment:
There's far more than that exposed - I don't think we should try to list it all.

Reply:
time series is already an incomplete list by itself. From experience, people care most about the three I now listed. I am 50/50 on this change, though.

Reply:
memory profiles, metadata, logs (at some point) - we'll keep on adding to this